utils/navigator_msgs/msg/Hosts.msg

```diff
@@ -0,0 +1 @@
+string[] hosts
```

Comment:
Also you should define the hostname constants here for use in the check hosts nodes to follow ros convention

In message file
```
string WAMV=mil-nav-wamv
```
In code
```
checkHost(Hosts.WAMV)
```
etc